docs/changes.rst

```diff
@@ -22,6 +22,30 @@ Released: not yet
 
 **Enhancements:**
 
+* Enhance output formats to allow an additional format group TEXT with
+  a single format ``text``. This format outputs the command result as a
+  text string to the console and is use for simple commands like
+  ``server interop`` that only output one piece of data. (see issue #594)
+
```

Comment:
The following four change items have been moved from the 0.6.0 release into the 0.7.0 release. I believe this was by accident. Please double check and undo the move.

Reply:
Actually two of them (from issue #594 and 598 are part of v 0.7.0 and I accidently put them into the section for 0.6.0 when I added them.  The other two were my mistake when I was moving the two that I had placed wrong.

**DONE**